ls1intum/Artemis
# Add notice when quiz result is missing and prevent NPE

### Checklist

### Description
- [x] I've added a missing result notice (see screenshot) that will be shown when we are after the *publishResultsDate* and there is no result for the quiz exercise
- [x] I've prevented a null pointer exception that will be shown in that case

### Steps for Testing
1. Log in to Artemis
2. Create an exam with at least one quiz exercise and a *publishResultsDate*
3. Participate in the exam
4. Wait until the *publishResultsDate* and go to the exam summary page (participation)
5. Check that there is no error on console any longer
6. Check that the blue alert box from the screenshot below is shown

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/87764672-0c09e700-c817-11ea-8dba-e804725e48e8.png)
